.travis.yml

```diff
@@ -51,50 +55,8 @@ jobs:
     - name: plugin install
       script: ./gradlew install
 
-    # AGP 3.4.0 E2E tests
-    - name: AGP 3.4.0 E2E tests
-      env:
-        - AGP_VERSION=3.4.0
-        - GRADLE_WRAPPER_VERSION=5.1.1
-        - RN_FIXTURE_DIR=features/fixtures/rn060/android
-      script: bundle exec maze-runner -c --verbose --fail-fast
-
-      # AGP 3.5.0 E2E tests
-    - name: AGP 3.5.0 E2E tests
-      env:
-        - AGP_VERSION=3.5.0
-        - GRADLE_WRAPPER_VERSION=5.4.1
-        - RN_FIXTURE_DIR=features/fixtures/rn060/android
-      script: bundle exec maze-runner -c --verbose --fail-fast
-
-      # AGP 3.6.0 E2E tests
-    - name: AGP 3.6.0 E2E tests
-      env:
-        - AGP_VERSION=3.6.0
-        - GRADLE_WRAPPER_VERSION=5.6.4
-        - RN_FIXTURE_DIR=features/fixtures/rn061/android
-      script: bundle exec maze-runner -c --verbose --fail-fast
-
-      # AGP 4.0.0 E2E tests
-    - name: AGP 4.0.0 E2E tests
-      env:
-        - AGP_VERSION=4.0.0
-        - GRADLE_WRAPPER_VERSION=6.1.1
-        - RN_FIXTURE_DIR=features/fixtures/rn062/android
-      script: bundle exec maze-runner -c --verbose --fail-fast
-
-      # AGP 4.1.0 E2E tests
-    - name: AGP 4.1.0 E2E tests
-      env:
-        - AGP_VERSION=4.1.0
-        - GRADLE_WRAPPER_VERSION=6.5.1
-        - RN_FIXTURE_DIR=features/fixtures/rn063/android
-      script: bundle exec maze-runner -c --verbose --fail-fast
-
-      # AGP 4.2.0 E2E tests
-    - name: AGP 4.2.0 E2E tests
+      # AGP E2E tests
+    - name: AGP E2E tests
       env:
-        - AGP_VERSION=4.2.0-beta03
```

Comment:
It might be nice to define these explicitly just so that it's obvious what is being used, and to make it easy to change in future from one place.

Reply:
We do also (sort of) have this in `env.rb`: https://github.com/bugsnag/bugsnag-android-gradle-plugin/pull/395/files#diff-3c1d97c5f1fa7ca90a4a96df18478ce19fe4d3a8dbaa3914ccc6ec6c43e81cbe